### ISSUE TITLE
macOS support builds but doesn't work

### ISSUE THREAD

**Issue by daminetreg** [header-only/inglued#2]
By merging #1 we made it compile for macOS, thanks to @linkineo, but it doesn't work for the moment, due to some paths issues.

It can be reproduced easily by cloning https://github.com/header-only/example-dependency and checking out the list-your-deps tag, then running `inglued seal` as presented below :  

```
Damiens-MacBook-Pro:inglued daminetreg$ cd example/example-dependency/
Damiens-MacBook-Pro:example-dependency daminetreg$ git checkout list-your-deps
Damiens-MacBook-Pro:example-dependency daminetreg$ ../../build/inglued seal
Fetching and glueing in your git repository direct dependencies:
	 fetching boostorg/fusion...libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: Cannot add the dependency, your working tree must not have changes to already commited files. Output is : 
 git fetch https://github.com/boostorg/fusion.git boost-1.62.0
warning: no common commits
From https://github.com/boostorg/fusion
 * tag               boost-1.62.0 -> FETCH_HEAD
error: Invalid path 'deps//fusion/.gitattributes'
error: Invalid path 'deps//fusion/doc/Jamfile'
error: Invalid path 'deps//fusion/doc/acknowledgements.qbk'
error: Invalid path 'deps//fusion/doc/adapted.qbk'
error: Invalid path 'deps//fusion/doc/algorithm.qbk'
error: Invalid path 'deps//fusion/doc/changelog.qbk'
error: Invalid path 'deps//fusion/doc/container.qbk'
error: Invalid path 'deps//fusion/doc/extension.qbk'
error: Invalid path 'deps//fusion/doc/functional.qbk'
error: Invalid path 'deps//fusion/doc/fusion.qbk'
error: Invalid path 'deps//fusion/doc/html/images/fusion_org.png'
error: Invalid path 'deps//fusion/doc/html/index.html'
```

It might be due to the passing of arguments to the bp::child, but might as well be something else. Have to investigate.

**Comment by linkineo** [comment]
Solved in #3. Test is now passing with the example above.

**Comment by daminetreg** [comment]
Indeed checked it's solved in #3 .
